packages/constants/src/lib/constants/endpoints.ts

```diff
@@ -0,0 +1,52 @@
+export enum LIT_ENDPOINT_VERSION {
+  V0 = '/',
+  V1 = '/v1',
+}
+
+export const LIT_ENDPOINT = {
```

Comment:
Could be beneficial to add a type annotation for this, so that any typos in this array are flagged by compiler here, rather than getting a 'path is type string | undefined' some where else in the code.

Reply:
You're right! will do that in a separate PR!